invesalius/data/visualization/probe_visualizer.py

```diff
@@ -0,0 +1,166 @@
+import os
+
+import vtk
+
+import invesalius.data.vtk_utils as vtku
+from invesalius import inv_paths
+from invesalius.pubsub import pub as Publisher
+
+
+class ProbeVisualizer:
+    """
+    A class for visualizing probe in the volume viewer.
+    """
+
+    def __init__(self, renderer, interactor):
+        self.renderer = renderer
+        self.interactor = interactor
+
+        self.probe_actor = None
+        self.probe_path = os.path.join(inv_paths.OBJ_DIR, "stylus.stl")
+        self.show_probe = False
+        self.is_navigating = False
+
+        self.AddProbeActor(self.probe_path)
+        self.__bind_events()
+
+    def __bind_events(self):
+        Publisher.subscribe(self.ShowProbe, "Show probe in viewer volume")
+        Publisher.subscribe(self.UpdateProbePose, "Update probe pose")
+        Publisher.subscribe(self.OnNavigationStatus, "Navigation status")
+
+    def OnNavigationStatus(self, nav_status, vis_status):
+        self.is_navigating = nav_status
+
+    def ShowProbe(self, state):
+        self.show_probe = state
+
+        if self.probe_actor:
+            self.probe_actor.SetVisibility(self.show_probe)
+            if not self.is_navigating:
+                self.interactor.Render()
```

Comment:
You could use Publisher.sendMessage("Render volume viewer") and remove the arg interactor